test/index.js

```diff
@@ -50,13 +50,13 @@ then use `geth attach ipc:///tmp/ethereum_dev_mode/geth.ipc` with
 var account = personal.newAccount('')
 account
 // and copy the account to below
-personal.unlockAccount(account)
+personal.unlockAccount(account, '', 1000000)
 miner.setEtherbase(account)
-miner.start()
+miner.start(n)
```

Comment:
Is `n` defined somewhere?

Reply:
no, by n i just meant pick a number.

Reply:
I would like to copy-paste the code, and in my geth, no argument is required.

Reply:
ok, ill just remove it